src/pybind/mgr/dashboard/services/feedback.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+import json
+import logging
+import os
+from qa.tasks.radosgw_admin import main
+
+from requests.auth import AuthBase
+
+from pybind.mgr.dashboard.rest_client import RestClient
+
+from .. import mgr
+from ..settings import Settings
+from ..exceptions import DashboardException
+from ..rest_client import RestClient
+
+class RedmineAuth(AuthBase):
+    def __init__(self, access_key):
+        self.access_key = str(access_key)
+
+    def __call__(self, r):
+        r.headers['X-Redmine-API-Key'] = self.access_key
+        return r
```

Comment:
Since I had to shift from RestClient to raw python requests for `create_issue()`, I was planning to do the same for `get_issues()` (if it's okay to not use the RestClient). Once that is done, we could remove the RedmineAuth part completely.  Right now, the RedmineAuth is called in init func and `get_api_key()` is called in create function. 

For now I'll use the `Setthings.CEPH..` here directly like you suggested. Pushing the changes.

Reply:
Ok!